client/app/templates/templates.service.js

```diff
@@ -94,4 +95,20 @@ export function TemplatesServiceFactory(CollectionsApi) {
 
     return CollectionsApi.post(collection, template.id, {}, editObj);
   }
+
+  function deleteTemplates(templates) {
+    const templateIds = [];
+    templates.forEach(function (template) {
```

Comment:
but really we shouldn't be using the services to process files, keep them as generic as possible, do this before you make the call, in the component